CRM/Custom/Import/Form/MapField.php

```diff
@@ -6,7 +6,7 @@
 class CRM_Custom_Import_Form_MapField extends CRM_Contact_Import_Form_MapField {
   protected $_parser = 'CRM_Custom_Import_Parser_Api';
   protected $_mappingType = 'Import Multi value custom data';
-  protected $_highlightedFields = array();
+//  protected $_highlightedFields = array();
```

Comment:
If you like to see this differently, I can change it of course. I'll fix the styling issues already